addons/binding/org.openhab.binding.modbus.sunspec/src/main/java/org/openhab/binding/modbus/sunspec/internal/block/CommonModelBlock.java

```diff
@@ -0,0 +1,114 @@
+/**
+ * Copyright (c) 2010-2019 Contributors to the openHAB project
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package org.openhab.binding.modbus.sunspec.internal.block;
+
+/**
+ * This class contains information parsed from the Common Model block
+ *
+ * @author Nagy Attila Gabor - Initial contribution
+ *
+ */
+public class CommonModelBlock extends AbstractSunSpecMessageBlock {
+
+    /**
```

Comment:
If possible, can you use immutable classes (everything set in the constructor, everything public final, no getters, no setters)? That helps for thread safety but also for reviewing, because less lines of boilerplate code.